website/pages/docs/commands/operator/migrate.mdx

```diff
@@ -65,6 +65,47 @@ storage_destination "consul" {
 }
 ```
 
+## Migrating to integrated raft storage
```

Comment:
If this will be our official migration documentation, could you add a pretty visible note about the current challenges in migrating away as well here?

Reply:
I had imagined that we would link to a separate migration guide